backend/tests/apps/owasp/index/registry/committee_test.py

```
@@ -0,0 +1,19 @@
+from unittest.mock import patch
+
+from apps.owasp.index.registry.committee import CommitteeIndex
+
+
+class TestCommitteeIndex:
+    def test_get_entities(self):
+        """Test get_entities returns active committees with select_related."""
+        with patch.object(CommitteeIndex, "__init__", lambda self: None):
```

Ruff (0.15.1)

[warning] Unused lambda argument: `self`

(ARG005)